### PR TITLE
Add input options related to `copy()`

### LINKED ISSUE #159: Allow absolute paths for the `build_dir` option
Could you please add an option to make this `path.join` optional?

https://github.com/crazy-max/ghaction-github-pages/blob/1ae239f7b822f4ef2d57ca2c7792713696508b10/src/main.ts#L66

The reason is that I would like to deploy a folder built with [Nix](https://nixos.org/). When Nix builds something, the output lands in a central store under `/nix/store` and the output which gets placed in the current directory is a symlink to the store location. Since the `build_dir` option needs a relative path, I end up having to work around this problem by prepending it with `../../../../../`, which is kind of ugly. It works for now, but I would prefer to be able to provide an absolute path (or even better, just be able to provide the symlink, but that doesn't seem to work because the `copy` function doesn't want to overwrite a folder with a symlink).

Thanks!

### PR DESCRIPTION
**Pull request by yrd**
This PR adds two new input options:

- **absolute_build_dir** treats the `build_dir` option as absolute, instead of a path relative to the current working directory (see #159)
- **follow_symlinks** will dereference symbolic links, allowing entire directory trees to be copied

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -29,6 +29,12 @@ inputs:
   build_dir:
     description: 'Build directory to deploy'
     required: true
+  absolute_build_dir:
+    description: 'Whether to treat build_dir as an absolute path'
+    required: false
+  follow_symlinks:
+    description: 'If enabled, the content of symbolic links will be copied'
+    required: false
```
Comment (crazy-max) [inline action.yml]:
```suggestion
  absolute_build_dir:
    description: 'Whether to treat build_dir as an absolute path'
    default: 'false'
    required: false
  follow_symlinks:
    description: 'If enabled, the content of symbolic links will be copied'
    default: 'false'
    required: false
```

**Review by crazy-max — CHANGES_REQUESTED** [review]
Minor nit otherwise lgtm

**Review by crazy-max — APPROVED** [review]
LGTM thanks!

**Comment by yrd** [comment]
Thank you!

**Pull request merged**